### PR TITLE
Validators.required value with only whitespace

### PR DESCRIPTION
**Pull request by arthurhamon**
A control value with only whitespace should be considered empty.
Add a trim on the function that check for empty values.

### CODE REVIEW

File: src/validators.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { fromPromise } from './utils'
 
 function isEmptyInputValue(value) {
-  return value == null || value.length === 0
+  return value == null || value.trim().length === 0
```
Comment (bietkul) [inline src/validators.js]:
```suggestion
  return value == null || (typeof value === 'string' && value.trim().length === 0)
```

**Review by bietkul — CHANGES_REQUESTED** [review]

**Comment by arthurhamon** [comment]
Request accepted :)

**Comment by arthurhamon** [comment]
Can you merge it and create a new release ?

**Comment by arthurhamon** [comment]
Hello @bietkul can we merge it ?

**Review by bietkul — APPROVED** [review]

**Pull request closed without merging**